translate/storage/yaml.py

```diff
@@ -171,18 +177,22 @@ def nested_set(target, path, value):
             default_flow_style=False, encoding='utf-8', allow_unicode=True
         ))
 
+    def _parse_dict(self, data, prev):
```

Comment:
If I am not wrong getting this block of code out to this new `_parse_dict` method doesn't change the behavior, isn't it? If the answer is yes I would prefer this cleanup to happen in a separate commit.

Reply:
I see now that the intention is to allow to override it on the subclass. Anyway IMHO it is cleaner if this happens in a separate commit.